odl/solvers/functional/default_functionals.py

```diff
@@ -1046,8 +1047,23 @@ def __init__(self, space, prior=None):
         space : `DiscreteLp` or `FnBase`
             Domain of the functional.
         prior : ``space`` `element-like`, optional
-            Data term, positive.
+            Depending on the context, the prior, target or data
+            distribution. It is assumed to be nonnegative.
             Default: if None it is take as the one-element.
+
+        Examples
+        --------
+        >>> space = odl.rn(3)
+        >>> prior = 3*space.one()
+        >>> func = KullbackLeibler(space, prior=prior)
```

Comment:
Do `func = odl.solvers.KullbackLeibler(space, prior=prior)`, changes nothing here but makes the example more instructive and copy-pastable for others